packages/next/package.json

```
@@ -76,6 +76,7 @@
     "babel-plugin-transform-async-to-promises": "0.8.10",
     "babel-plugin-transform-react-remove-prop-types": "0.4.24",
     "chalk": "2.4.2",
+    "compression": "^1.7.4",
```

Let's pin this version to match all other deps.
```suggestion
    "compression": "1.7.4",
```